addons/web/static/src/legacy/js/views/list/list_renderer.js

```diff
@@ -983,6 +983,45 @@ var ListRenderer = BasicRenderer.extend({
             $tr.addClass('o_list_no_open');
         }
         this._setDecorationClasses($tr, this.rowDecorations, record);
+        /**
+         * If the record is allowed to be opened and the current view is a list
+         * view (not a list embedded in a form view), add event listeners for
+         * Ctrl+Click and Middle Click to open the record in a new tab.
+         */
+        if (!$tr.hasClass('o_list_no_open')) {
+            $tr.mousedown(function(e){
+                var startURI;
+                var endURI;
+                var baseURI = $tr[0].baseURI
+                switch(e.originalEvent.which) {
+                    case 1:
+                    if (e.ctrlKey && baseURI.substring(baseURI.length - 4) === "list") {
+                        // Prevent record from being opened in current tab
+                        e.preventDefault();
+                        // Construct form URL for selected record
+                        [startURI, endURI] = baseURI.split("#")
+                        window.open(
+                            startURI + "#id=" + record.res_id + "&" +
+                            endURI.substring(0, endURI.length - 4) + "form",
+                            "_blank"
+                        )
+                    }
+                    break;
```

Comment:
Again, this might be a me thing, but I used to like to indent the code in a `case` to make it more visible at a glance.  Also don't forget your `;`s:   :wink: 
```suggestion
                    case 1:
                        if (e.ctrlKey && baseURI.substring(baseURI.length - 4) === "list") {
                            // Prevent record from being opened in current tab
                            e.preventDefault();
                            // Construct form URL for selected record
                            [startURI, endURI] = baseURI.split("#");
                            window.open(
                                startURI + "#id=" + record.res_id + "&" +
                                endURI.substring(0, endURI.length - 4) + "form",
                                "_blank"
                            );
                        }
                        break;
```

Reply:
I agree, the indentation looks better

Reply:
BTW, I'm not trying to make anyone follow that style.  Just pointing it out for discussion.